src/traces/indicator/attributes.js

```diff
@@ -0,0 +1,408 @@
+            'Sets the number to be displayed.'
+        ].join(' ')
+    },
+    align: {
+        valType: 'enumerated',
+        values: ['left', 'center', 'right'],
+        role: 'info',
+        editType: 'plot',
+        description: [
+            'Sets the horizontal alignment of the `text` within the box.',
+            'Note that this attribute has no effect if an angular gauge is displayed:',
+            'in this case, it is always centered'
+        ].join(' ')
+    },
+    // position
+    domain: domainAttrs({name: 'indicator', trace: true, editType: 'calc'}),
+
+    vmin: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        dflt: 0,
+        description: [
+            'Sets the minimum value of the gauge.'
+        ].join(' ')
+    },
+    vmax: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        description: [
+            'Sets the maximum value of the gauge.'
+        ].join(' ')
+    },
+
+    title: {
+        text: {
+            valType: 'string',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the title of this indicator.'
+            ].join(' ')
+        },
+        align: {
+            valType: 'enumerated',
+            values: ['left', 'center', 'right'],
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the horizontal alignment of the title.',
+                'It defaults to `center` except for bullet charts',
+                'for which it defaults to right.'
+            ].join(' ')
+        },
+        font: extendFlat({}, textFontAttrs, {
+            description: [
+                'Set the font used to display the title'
+            ].join(' ')
+        }),
+        editType: 'plot'
+    },
+    number: {
+        valueformat: {
+            valType: 'string',
+            dflt: '.3s',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the value formatting rule using d3 formatting mini-language',
+                'which is similar to those of Python. See',
+                'https://github.com/d3/d3-format/blob/master/README.md#locale_format'
+            ].join(' ')
+        },
+        font: extendFlat({}, textFontAttrs, {
+            description: [
+                'Set the font used to display main number'
+            ].join(' ')
+        }),
+        suffix: {
+            valType: 'string',
+            dflt: '',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets a suffix appearing next to the number.'
+            ].join(' ')
+        },
+        editType: 'plot'
+    },
+    delta: {
+        reference: {
+            valType: 'number',
+            role: 'info',
+            editType: 'calc',
+            description: [
+                'Sets the reference value to compute the delta.'
```

Comment:
Maybe add something in the `description` about what happens when `reference` isn't set? 

In the case, it uses the "last" value as reference, correct?

Reply:
When it isn't set, it defaults to the current value.

Reply:
Great, is there a test locking down that behaviour?

Reply:
Done in e6d8525ab07c56ca10bdbac6762b06d1d57bf830 with test to :lock: down the behavior